carryingCapPlot.py

```diff
@@ -0,0 +1,38 @@
+import json
+import os
+import matplotlib.pyplot as plt
+
+def loadPopulationLogs(directory):
+    populationLogs = []
+
+    for filename in os.listdir(directory):
+        if filename.startswith("population_log_") and filename.endswith(".json"):
+            with open(os.path.join(directory, filename), 'r') as f:
+                populationLogs.append(json.load(f))
+
+    return populationLogs
+
+def plotPopulationOverTime(populationLogs):
+    carryingCapacities = []
+    
+    for log in populationLogs:
+        plt.plot(log, label=f'Seed {populationLogs.index(log) + 1}')
+        carryingCapacities.append(max(log))  # assuming the carrying capacity is the max population reached

```

Comment:
In `runSimulationAndLog` you consider the carrying capacity to be roughly the last ten values of the list. Maybe you can use the same logic here for runs that initially overshoot their carrying capacity?